perly.tab

```diff
@@ -432,27 +432,6 @@ yysymbol_name (yysymbol_kind_t yysymbol)
 }
 #endif
 
-#ifdef YYPRINT
-/* YYTOKNUM[NUM] -- (External) token number corresponding to the
-   (internal) symbol number NUM (which must be that of a token).  */
-static const yytype_int16 yytoknum[] =
-{
-       0,   256,   257,   258,   259,   260,   261,   262,   263,   264,
-     265,   266,   267,   268,   269,   270,   271,   272,   273,   274,
-     275,   276,   277,   278,   279,   280,   281,   282,   283,   284,
-     285,   286,   287,   288,   289,   290,   291,   292,   293,   294,
-     295,   296,   297,   298,   299,   300,   301,   302,   303,   304,
-     305,   306,   307,   308,   309,   310,   311,   312,   313,   314,
-     315,   316,   317,   318,   319,   320,   321,   322,   323,   324,
-     325,   326,   327,   328,   329,   330,   331,   332,   333,   334,
-     335,   336,   337,   338,   339,   340,   341,   342,   343,   344,
-     345,   346,   347,   348,   349,   350,   351,   352,   353,   354,
-     355,   356,   357,   358,   359,   360,   361,   362,   363,   364,
-     365,   366,   367,   368,   369,   370,   371,   372,   373,   374,
-     375,   376,   377,   378,   379,   380,   381,   382
-};
-#endif
-
```

Comment:
This is the only significant change I see. `yytoknum` is only used once in `perly.c`, also behind an `#ifdef YYPRINT` guard. Maybe that code should be removed, too?

The bison documentation (https://www.gnu.org/software/bison/manual/html_node/The-YYPRINT-Macro.html) says the `YYPRINT` feature was obsoleted by the introduction of the `%printer` directive in bison 1.50. I don't think we've ever used it in perl.

Reply:
It could be used to improve some of the debugging output, but as you say, we haven't used it, and it's deprecated.